logical/plugin/pb/translation.go

```diff
@@ -544,6 +551,15 @@ func ProtoAuthToLogicalAuth(a *Auth) (*logical.Auth, error) {
 		return nil, err
 	}
 
+	var boundCIDRs []*sockaddr.SockAddrMarshaler
+	for _, cidr := range a.BoundCidrs {
```

Comment:
Can use the parseutil function here too.

Reply:
https://github.com/hashicorp/vault/pull/4546